books/RayTracingInOneWeekend.html

```diff
@@ -106,8 +106,23 @@
      corner to be yellow.
 
 <div class='together'>
-Opening the output file (in ToyViewer on my mac, but try it in your favorite viewer and google “ppm
-viewer” if your viewer doesn’t support it) shows:
+Because the file is written to the program output, you'll need to redirect it to the image file.
+Typically this is done from the command-line by using the `>` redirection operator, like so:
+
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    build\Release\inOneWeekend.exe > image.ppm
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This is how things would look on Windows. On Mac or Linux, it would look like this:
+
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    build/Release/inOneWeekend > image.ppm
```

Comment:
the cmake output doesn't qualify as release. It just dumps the file name in the build folder
e.g.
```   
build/inOneWeekend > image.ppm
```